libkbfs/block_journal.go

```diff
@@ -725,39 +754,49 @@ func (be blockEntriesToFlush) flushNeeded() bool {
 // Only entries with ordinals less than the given ordinal (assumed to
 // be <= latest ordinal + 1) are returned.
 func (j *blockJournal) getNextEntriesToFlush(
-	ctx context.Context, end journalOrdinal) (
-	entries blockEntriesToFlush, err error) {
+	ctx context.Context, end journalOrdinal, maxToFlush int) (
+	entries blockEntriesToFlush, maxMDRevToFlush MetadataRevision, err error) {
 	first, err := j.j.readEarliestOrdinal()
 	if os.IsNotExist(err) {
-		return blockEntriesToFlush{}, nil
+		return blockEntriesToFlush{}, MetadataRevisionUninitialized, nil
```

Comment:
Hmm this seems dangerous. If we have both block and MD ops, and we push all block ops and then get interrupted, then the next run would just stall until the next operation, right?

We probably should return `maxMetadataRevision` in this case, or signal that some other way. Does that sound right?